src/client.js

```diff
@@ -94,6 +94,27 @@ function Client(manager, name, config) {
 
 	if (typeof client.config.sessions !== "object") {
 		client.config.sessions = {};
+	} else {
+		// TODO: This is just for backwards compatibility
```

Comment:
Do you want to add a note saying "Remove this in v3" or something?

---

On an unrelated note that, we've kept our deprecated stuff for a while, because we knew v3 was coming soon-ish, but in the future, I have nothing against deprecating things for a couple versions of a few months and then removing the deprecated support as part of a minor version release.
We've mentioned this in the past, but we're not following semver (we've even removed that from our changelogs): we're an end-user project and not a library, so we're not typically installed in `package.json`s with a `~` range or something, and we document these things pretty well in the changelogs.